### PR TITLE
Prac 07 feedback

### PR DESCRIPTION
**Pull request by Amarachukwu-Stella-Nneji**
please review my code @Ya-Jun-Liu

### CODE REVIEW

File: prac_07/guitar.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def __init__(self, name="", year=0, cost=0.0):
         self.cost = cost
 
     def __str__(self):
-        """Return name, year and cost."""
+        """Return name, year and estimate."""
         return f"{self.name} ({self.year}) : ${self.cost:.2f}"
 
     def get_age(self):
```
Comment (Ya-Jun-Liu) [inline prac_07/guitar.py:24]: Well done! It would be good to have a full stop at the end of the docstring.

File: prac_07/myguitars.py [hunk 2]
```diff
@@ -13,5 +13,18 @@ def main():
     for guitar in guitars:
         print(f"{guitar.name} ({guitar.year}) : {guitar.cost:,.2f}")
```
Comment (Ya-Jun-Liu) [inline prac_07/myguitars.py:14]: Nice work! I would personally print the list in a neat way.

File: prac_07/project_management.py [hunk 3]
```diff
@@ -46,6 +48,7 @@ def load_project(file_name):
 
 
 def display_projects(projects):
+    """Display project"""
     incomplete_projects = [project for project in projects if project.completion < 100]
     completed_projects = [project for project in projects if project.completion == 100]
```
Comment (Ya-Jun-Liu) [inline prac_07/project_management.py:54]: The program does not print the list when I choose menu option D for display.

File: prac_07/project_management.py [hunk 4]
```diff
@@ -36,6 +37,7 @@ def main():
```
Comment (Ya-Jun-Liu) [inline prac_07/project_management.py:37]: When 'F' is entered, it does nothing but print the menu again.

File: prac_07/project_management.py [hunk 5]
```diff
@@ -96,6 +102,7 @@ def update_project(projects):
 
 
 def filter_projects_by_date(projects):
+    """Filter project"""
     print("Projects filtered by date:")
     date_string = input("Enter the date (dd/mm/yyyy) to filter by: ")
     try:
```
Comment (Ya-Jun-Liu) [inline prac_07/project_management.py:108]: Nice error-checking

**Comment by Ya-Jun-Liu** [comment]
Overall, you have done some amazing work! However, I copied the project_management code from the master branch and ran it, and most functions seemed non-functional. I am sorry about that.

File: prac_07/project_management.py (hunk 3 above)
Comment (Amarachukwu-Stella-Nneji) [reply prac_07/project_management.py:54]: thats because the projects  aren’t loaded at the beginning of the program

**Comment by Amarachukwu-Stella-Nneji** [comment]
thanks for the feedbacks, i'll work on the program again

**Pull request merged**